src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -183,7 +184,19 @@ export class HaIntegrationCard extends LitElement {
                         >
                       `
                     : ""}
-                  ${devices.length && entities.length
+                  ${services.length
+                    ? html`
+                        <a
+                          href=${`/config/devices/dashboard?historyBack=1&config_entry=${item.entry_id}`}
+                          >${this.hass.localize(
+                            "ui.panel.config.integrations.config_entry.services",
+                            "count",
+                            services.length
+                          )}</a
+                        >
+                      `
+                    : ""}
+                  ${(devices.length || services.length) && entities.length
                     ? this.hass.localize("ui.common.and")
                     : ""}
```

Comment:
We should also render an `and` when there is both a service and a device.

Reply:
I had that first, but if the integration has both you end up with:
"x services and x devices and x entities" which feels weird :shrug: 
Add a comma maybe?

Reply:
yeah:

`2 devices, 1 service and x entities`